sale_manual_delivery/wizard/manual_proc_view.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+
+  <record id='manual_delivery_wizard_form' model='ir.ui.view'>
+    <field name='manual_delivery_wizard_form'/>
+    <field name='model'>manual.delivery</field>
+    <field name='arch' type='xml'>
+
+      <form string="Create Manually Delivery" name="manual_procurment_form">
+        <sheet>
+          <group>
+            <field name='date_planned' required="1"/>
+            <field name='order_id' invisible="1"/>
+          </group>
+          <group>
+            <field name='carrier_id'/>
+          </group>
+          <field name='line_ids'>
+            <tree editable="bottom" create="0">
+              <field name='manual_delivery_id' invisible="1"/>
+              <field name='order_line_id' invisible="1"/>
+              <field name='product_id'/>
+              <field name='ordered_qty'/>
+              <field name='existing_qty'/>
+              <field name='remaining_qty'/>
+              <field name='to_ship_qty'/>
+            </tree>
+          </field>
+        </sheet>
+        <footer>
+          <button name='record_picking'
+                  string='Create Picking'
+                  class='btn-primary'
+                  type='object'/>
+          <button string="Cancel" class="btn-default" special="cancel" />
+        </footer>
+      </form>
+
+    </field>
+  </record>
+
+  <act_window id='action_wizard_manual_delivery'
+              name='Create Manual Delivery'
```

Comment:
I'm not sure if this is good. I'm getting a singleton error when I select more than one SO in the list view and then I click on the action.